build/dependencies.props

```diff
@@ -3,10 +3,10 @@
     <BenchmarkDotNetPackageVersion>0.11.3</BenchmarkDotNetPackageVersion>
     <CommandLineParserPackageVersion>2.3.0</CommandLineParserPackageVersion>
     <GoogleProtobufPackageVersion>3.11.4</GoogleProtobufPackageVersion>
-    <GrpcDotNetPackageVersion>2.27.0-pre1</GrpcDotNetPackageVersion>  <!-- Only use for example projects -->
+    <GrpcDotNetPackageVersion>2.28.0-pre2</GrpcDotNetPackageVersion>  <!-- Only use for example projects -->
```

Comment:
FYI, this is going to conflict with https://github.com/grpc/grpc-dotnet/pull/855.

Reply:
They're different. This is fine.